### ISSUE TITLE
finding matlabengine.dll/.so when built with SPM

### ISSUE THREAD

**Issue by KrisThielemans** [SyneRBI/SIRF-SuperBuild#458]
@rijobro's wrapper for SPM uses the MATLAB Engine to call Matlab from C++.

He created #366  for Linux as at some point, the relevant  .so wasn't find, although the [last comment says](https://github.com/SyneRBI/SIRF-SuperBuild/pull/366#issuecomment-605974091)
> Hold off merging this one. Seems unnecessary for my current build, not sure what I've changed.

I have exactly the same problem on Windows. I need to add `;c\Program Files\MATLAB\R2020b\bin\win64;c:\Program Files\MATLAB\R2020b\extern\bin\win64` to my `PATH`. Otherwise, the `Reg` tests refuse to run (sadly, they don't tell you why. It's only when I ran them with
```
start .\src\Registration\cReg\tests\RelWithDebInfo\REG_TEST_CPLUSPLUS.exe
```
that Windows told me why the test failed.

**Comment by rijobro** [comment]
I would merge #366 if I were you, it's fairly harmless since the `.so`'s are in their own folder. I guess it would be pretty straightforward to do the same for windows, too.

**Comment by KrisThielemans** [comment]
I've merged #366, and added the relevant lines in the template at #460

**Comment by KrisThielemans** [comment]
Closing as it will be closed with #460
